diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
index fc52033..1baaf61 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
@@ -1,5 +1,5 @@
 import invariant from 'invariant';
-import React, {
+import {
   forwardRef,
   memo,
   useCallback,
@@ -215,7 +215,12 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
     const animatedCurrentIndex = useReactiveSharedValue(
       animateOnMount ? -1 : _providedIndex
     );
-    const animatedPosition = useSharedValue(Dimensions.get('window').height);
+  const animatedPosition = useSharedValue(
+  Platform.select({
+    web: Dimensions.get('window').height,
+    default: Dimensions.get('screen').height,
+  }) + bottomInset
+);
 
     // conditional
     const didAnimateOnMount = useSharedValue(
